LINK/etc/profile.d/evm.sh

```diff
@@ -4,3 +4,16 @@
 alias ap='/usr/bin/appliance_console'
 alias vmdb='cd /var/www/miq/vmdb'
 alias appliance='[[ -n ${APPLIANCE_SOURCE_DIRECTORY} ]] && cd ${APPLIANCE_SOURCE_DIRECTORY}'
+
+# Tail MIQ Logs:
+function tailmiq()
+{
+  tail -f /var/www/miq/vmdb/log/$1.log
```

Comment:
> Regardless, I think we need more than just `evm.log`

@mikedahlgren I'm in agreement with you.

Not sure if you are familiar with `${var-default}`:

```bash
ret="${var-default}"
# equivalent to:
if [ "${var}" == "" ] ; then
  ret="default"
else
  ret="${var}"
fi
```


So if you use `${1-evm}`, it basically lets the user provide the log file, but defaults to evm it it was not provided (rather than the existing code that will blow up)